src/applications/personalization/profile-2/tests/components/personal-information/PersonalInformation.delete-address.unit.spec.jsx

```diff
@@ -0,0 +1,272 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { waitForElementToBeRemoved } from '@testing-library/react';
+import { expect } from 'chai';
+import { setupServer } from 'msw/node';
+
+import { resetFetch } from 'platform/testing/unit/helpers';
+import { FIELD_TITLES, FIELD_NAMES } from 'vet360/constants';
+
+import * as mocks from '../../../msw-mocks';
+import PersonalInformation from '../../../components/personal-information/PersonalInformation';
+
+import {
+  createBasicInitialState,
+  renderWithProfileReducers,
+  wait,
+} from '../../unit-test-helpers';
+
+const ui = (
+  <MemoryRouter>
+    <PersonalInformation />
+  </MemoryRouter>
+);
+let view;
+let server;
+
+function getEditButton(addressName) {
+  let editButton = view.queryByText(new RegExp(`add.*${addressName}`, 'i'), {
+    selector: 'button',
+  });
+  if (!editButton) {
+    // Need to use `queryByRole` since the visible label is simply `Edit`, but
+    // the aria-label is more descriptive
+    editButton = view.queryByRole('button', {
+      name: new RegExp(`edit.*${addressName}`, 'i'),
+    });
+  }
+  return editButton;
+}
+
+function deleteAddress(addressName) {
+  getEditButton(addressName).click();
```

Comment:
Just a general thought, not related to this block per se - it feels weird to me that the file extension is `unit.spec.jsx` while we're doing integration tests.. 

I do really appreciate your thoroughness with the tests, there just seems a bit of a mismatch if you know what I mean 😄 I don't really know of an action item here though

Reply:
Ha, yeah I think that's required by our `test:unit` script. Remember when we say "unit test" that just means "mocha test" 😁